examples/api/lib/material/menu_anchor/menu_anchor.0.dart

```diff
@@ -7,7 +7,22 @@
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 
-void main() => runApp(const MenuApp());
+void main() {
```

Comment:
nit (since you reordered this file anyways): How about moving the main and the MenuApp class to the end of the file? Those are arguably the least interesting aspects of this sample, but they appear "above the fold" when this is shown within an embedded dartpad in the API docs. Might be cool if something more relevant shows up there.

(don't feel super-strongly about this, just an idea)